src/wutils_db.erl

```diff
@@ -90,8 +95,10 @@ query(Query, PG) ->
             V
     catch
         exit:connection_timeout ->
+            close(PG),
             error({pgsql, connection_timeout, PG});
```

Comment:
If you close PG first, would you still get the error line sending the message containing the PG value?

Reply:
Yes: Erlang variables are immutable.

Reply:
yes, it would print the same

Reply:
Longer explanation:
`PG` is a "Port", which is the pid of a process holding something like a filehandle. It might also have some metadata, like `{pg_connection, Pid}`.
When you close it, you contact the process and ask it to close the filehandle and then terminate. But the pid itself remains and that's all that the stack trace is going to print. 

Try at home:

```
$ erl
Erlang/OTP 20 [erts-9.1.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
Eshell V9.1.2  (abort with ^G)
1> {ok, File} = file:open("/tmp/hello.txt", [write]).
{ok,<0.71.0>}
2> file:close(File).
ok
3> File.
<0.71.0>
```